### PR TITLE
Added Social Media Links as well as APIs.

### PR DESCRIPTION
**Pull request by nzec**
I have also added APIs for Flickr and Twitter linked to FOSSASIA's accounts.

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-gci17.fossasia.org
```
Comment (abishekvashok) [inline CNAME]: This is a required file as this site is hosted on github (gh-pages)

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please try with the current font included in the theme, a suggestion only though

File: CNAME (hunk 1 above)
Comment (nzec) [reply CNAME]: Oh okay. I'll get that back!

Comment (nzec) [reply CNAME]: Done

**Comment by nzec** [comment]
And what should I do with the font?

**Comment by abishekvashok** [comment]
@duttaditya18 sticking with the current font would be good, i guess.

**Comment by nzec** [comment]
I don't think I changed any font. @abishekvashok  It is the same font as there is in the original.

File: css/font-awesome.css [hunk 2]
```diff
@@ -0,0 +1,2337 @@
+/*!
```
Comment (abishekvashok) [inline css/font-awesome.css]: I was mentioning this file

Comment (nzec) [reply css/font-awesome.css]: That is for symbols. (FontAwesome delivers logos and symbols as fonts) FontAweosme which was earlier linked was outdated, so I linked the newer one. Nothing special changed other than the Google+ logo.

**Comment by nzec** [comment]
@abishekvashok Can this pull request be merged now?

**Comment by nzec** [comment]
@abishekvashok Also, who will accept my contribution on Code-In? How am I supposed to do other tasks if this one does not get approved? Sorry for bothering you too much!

**Review by poush — CHANGES_REQUESTED** [review]
Update only current minified font awesome CSS file. Do not add a new one.

**Comment by abishekvashok** [comment]
> @abishekvashok Also, who will accept my contribution on Code-In? How am I supposed to do other tasks if this one does not get approved? Sorry for bothering you too much!

Some mentor would merge your code, after you submit a task for review if you have met all conditions the task will be accepted and then you can go for other tasks. :) You will also get peer reviews just like I did.

**Comment by akarshan96** [comment]
Hi, @duttaditya18 can you please squash the commits and give relevant commit message to the single squashed commit. Rest looks fine 👍

**Comment by nzec** [comment]
@poush Updated the minified FontAwesome CSS file only and the fonts.
@akarshan96 Squashed the commits into 94c9ac9

**Comment by nzec** [comment]
@poush @akarshan96 @abishekvashok  Just tell me what should be done next. I have done everything that was asked.

**Comment by abhinavk96** [comment]
@duttaditya18 You have not squashed the commits at all :) as you can see your PR contains 12 commits.

**Comment by nzec** [comment]
@CosmicCoder96 How do I squash it using GitKraken?

**Pull request closed without merging**